docs/source/conf.py

```diff
@@ -0,0 +1,256 @@
+
+latex_elements = {
+    # The paper size ('letterpaper' or 'a4paper').
+    #
+    # 'papersize': 'letterpaper',
+    # The font size ('10pt', '11pt' or '12pt').
+    #
+    # 'pointsize': '10pt',
+    # Additional stuff for the LaTeX preamble.
+    #
+    # 'preamble': '',
+    # Latex figure (float) alignment
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, "eo-grow.tex", doc_title, author, "manual"),
+]
+
+# -- Options for manual page output ---------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [(master_doc, "eo-grow", doc_title, [author], 1)]
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, "eo-grow", doc_title, author, "eo-grow", "One line description of project.", "Miscellaneous"),
+]
+
+# -- Options for Epub output ----------------------------------------------
+
+# Bibliographic Dublin Core info.
+epub_title = project
+epub_author = author
+epub_publisher = author
+epub_copyright = project_copyright
+
+# The unique identifier of the text. This can be a ISBN number
+# or the project homepage.
+#
+# epub_identifier = ''
+
+# A unique identification for the text.
+#
+# epub_uid = ''
+
+# A list of files that should not be packed into the epub file.
+epub_exclude_files = ["search.html"]
+
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {"https://docs.python.org/3.8/": None}
+
+# copy examples
+# try:
+#     shutil.copytree("../../examples", "./examples")
+# except FileExistsError:
+#     pass
+
+
+MARKDOWNS_FOLDER = "./markdowns"
+shutil.rmtree(MARKDOWNS_FOLDER, ignore_errors=True)
+os.mkdir(MARKDOWNS_FOLDER)
+
+
+def process_readme():
+    """Function which will process README.md file and create INTRO.md"""
+    with open("../../README.md", "r") as file:
+        readme = file.read()
+
+    readme = readme.replace("[`", "[").replace("`]", "]").replace("docs/source/", "")
+    readme = readme.replace("**`", "**").replace("`**", "**")
+
+    chapters = [[]]
+    for line in readme.split("\n"):
+        if line.strip().startswith("## "):
+            chapters.append([])
+        if line.startswith("<img"):
+            line = "<p></p>"
+
+        chapters[-1].append(line)
+
+    chapters = ["\n".join(chapter) for chapter in chapters]
+
+    intro = "\n".join(
+        [
+            chapter
+            for chapter in chapters
+            if not (chapter.startswith("## Install") or chapter.startswith("## Documentation"))
```

Comment:
In `eo-learn` we exclude `Install` chapter because we create a separate page for it. Is this also planned here or should we leave the `Install` chapter on this page for now?

Reply:
decided to keep it for now, you're right